README.md

```diff
@@ -29,23 +40,23 @@ and some other logging and metadata.
 API Definition
 --------------
 
-See the human-readable [Reference Documentation](https://ga4gh.github.io/task-execution-schemas/docs/)
-You can also explore the specification in
-the [Swagger Editor](https://editor.swagger.io/?url=https://ga4gh.github.io/task-execution-schemas/openapi.yaml)**
-*Manually load the JSON if working from a non-develop branch version.* Preview documentation from
-the [gh-openapi-docs](https://github.com/ga4gh/gh-openapi-docs) for the development
-branch [here](https://ga4gh.github.io/task-execution-schemas/preview/develop/docs/index.html)
+See the human-readable [Reference Documentation](https://ga4gh.github.io/task-execution-schemas/docs/).
 
-> All documentation and pages hosted at 'ga4gh.github.io/task-execution-schemas' reflect the latest API release from
-> the `master` branch. To monitor the latest development work, add 'preview/\<branch\>' to the URLs above (e.g., '
-> ga4gh.github.io/task-execution-schemas/preview/\<branch\>/docs'). To view the latest *stable*
-> development API specification, refer to the `develop` branch.
+
+You can also examine the specification in the [Swagger Editor](https://editor.swagger.io/?url=https://ga4gh.github.io/task-execution-schemas/openapi.yaml).  
+
+*If you are using a version from a non-develop branch, please load the JSON manually.* You can preview the documentation from the [gh-openapi-docs](https://github.com/ga4gh/gh-openapi-docs) for the development branch [here](https://ga4gh.github.io/task-execution-schemas/preview/develop/docs/index.html).
```

Comment:
I would probably remove the

> [...] from the [gh-openapi-docs](...) [...]

part, because the average user doesn't need to check out that branch (or even know about it). A reference to the built docs for the development branch should be enough.

Reply:
Also, you can merge that statement with the next section as an example for how people can get branch-specific previews of the docs.